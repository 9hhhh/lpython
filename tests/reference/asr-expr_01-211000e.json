{
    "basename": "asr-expr_01-211000e",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/expr_01.py",
    "infile_hash": "4998ff229ad08556d7d216d3ad94d7eb069c14dc45d930a839c4ffc8",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-expr_01-211000e.stdout",
    "stdout_hash": "f4f91c6e410b8c81fc1bec4381bf714c268e9487f7c6fb7980f4303e",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}