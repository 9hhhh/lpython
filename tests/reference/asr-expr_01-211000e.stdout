(TranslationUnit (SymbolTable 1 {main: (Function (SymbolTable 2 {main_return_var: (Variable 2 main_return_var ReturnVar () () Default (Integer 8 []) Source Public Required .false.), x: (Variable 2 x Local () () Default (Integer 4 []) Source Public Required .false.), x2: (Variable 2 x2 Local () () Default (Integer 8 []) Source Public Required .false.), y: (Variable 2 y Local () () Default (Real 4 []) Source Public Required .false.), y2: (Variable 2 y2 Local () () Default (Real 8 []) Source Public Required .false.)}) main [] [(= (Var 2 x) (BinOp (BinOp (ConstantInteger 2 (Integer 4 [])) Add (ConstantInteger 3 (Integer 4 [])) (Integer 4 []) (ConstantInteger 5 (Integer 4 [])) ()) Mul (ConstantInteger 5 (Integer 4 [])) (Integer 4 []) (ConstantInteger 25 (Integer 4 [])) ()) ()) (= (Var 2 x) (BinOp (ConstantInteger 2 (Integer 4 [])) Pow (ConstantInteger 3 (Integer 4 [])) (Integer 4 []) (ConstantInteger 8 (Integer 4 [])) ()) ()) (= (Var 2 x) (BinOp (ConstantInteger 54 (Integer 4 [])) Sub (ConstantInteger 100 (Integer 4 [])) (Integer 4 []) (ConstantInteger -46 (Integer 4 [])) ()) ()) (= (Var 2 x2) (BinOp (BinOp (ConstantReal 3.454000 (Real 8 [])) Sub (ConstantReal 765.430000 (Real 8 [])) (Real 8 []) (ConstantReal -761.976000 (Real 8 [])) ()) Add (ConstantReal 534.600000 (Real 8 [])) (Real 8 []) (ConstantReal -227.376000 (Real 8 [])) ()) ()) (= (Var 2 x2) (BinOp (ConstantReal 5346.565000 (Real 8 [])) Mul (ConstantReal 3.450000 (Real 8 [])) (Real 8 []) (ConstantReal 18445.649250 (Real 8 [])) ()) ()) (Print () [(Var 2 x)]) (Print () [(Var 2 x2)]) (= (Var 2 main_return_var) (ConstantInteger 0 (Integer 8 [])) ())] (Var 2 main_return_var) Source Public Implementation ())}) [])
